src/components/auth/socialAuth.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { connect } from "react-redux";
+import { facebookAuth, googleAuth } from "../../actions/authActions/socialAuth";
+import { Redirect } from "react-router-dom";
+import PropTypes from "prop-types";
+
+export class SocialAuth extends Component {
+  onFailure = error => {
+    alert(error);
+  };
+  handleFacebookResponse = response => {
+    this.props.facebookAuth({ access_token: response.accessToken });
+  };
+
+  handleGoogleResponse = response => {
+    this.props.googleAuth({ access_token: response.accessToken });
+  };
+  render() {
+    if (this.props.isLoggedIn) {
+      return <Redirect to="/" />;
+    }
+    return (
+      <div>
+        <FacebookLogin
+          appId={process.env.REACT_APP_APP_ID}
```

Comment:
Let's have a different file called config in src and put all the process.env.VARIABLES in there and import, and call the variables in all files needed.